docs/task/task.md

```diff
@@ -104,6 +104,44 @@ class DispatcherMixin:
         return cls.tcp_port
 ```
 
+
+## Get command outputs through git repository
+
+Raw command outputs stored in a Git repository can be used in scenarios where Nautobot is not able to connect directly to the network devices. This is useful for disconnected or air-gapped environments, lab setups, or testing purposes.
+
+The feature is integrated into the `NetmikoDefault` dispatcher and is controlled by the `offline_commands` setting with the following precedence:
+
+1. `obj.cf["offline_commands"]` — if it exists and is a valid boolean value.
+2. `obj.get_config_context()["offline_commands"]` — if it exists and is a valid boolean value.
+3. `cls.offline_commands` — the default class attribute defined in `NetmikoDefault`, which defaults to `False`.
+
+When enabled, the dispatcher attempts to read the expected command output from the filesystem instead of executing the command on a live device. This requires the output files to be named in a filesystem-safe format.
```

Comment:
```suggestion
When enabled, the dispatcher attempts to read the expected command output from the filesystem (via the keyword arguments git_repo_obj, commands_folder_name, and command_relative_path) instead of executing the command on a live device. This requires the output files to be named in a filesystem-safe format.
```